packages/bot/src/components/chart/chart.jsx

```diff
@@ -0,0 +1,132 @@
+    // );
+
+    render() {
+        const {
+            show_digits_stats,
+            main_barrier,
+            // should_refresh,
+            // resetRefresh,
+            chart_type,
+            is_mobile,
+            granularity,
+            wsSendRequest,
+            wsForget,
+            wsForgetStream,
+            wsSubscribe,
+            settings,
+            symbol,
+            is_socket_opened,
+
+        } = this.props;
+
+        const barriers = main_barrier ? [main_barrier] : [];
+        // smartcharts only way to refresh active-symbols is to reset the connection.
+        // const is_socket_opened = this.props.is_socket_opened && !should_refresh;
+
+        // if (should_refresh) {
+        //     setImmediate(() => resetRefresh());
+        //     // TODO: fix this in smartcharts, it should be possible to update
+        //     // active-symbols without re-rendering the entire chart.
+        //     return null;
+        // }
+
+        return (
+            <SmartChart
+                id='dbot'
+                barriers={barriers}
+                // bottomWidgets={ show_digits_stats ? this.bottomWidgets : null}
+                showLastDigitStats={show_digits_stats}
+                chartControlsWidgets={this.chartControlsWidgets}
+                chartStatusListener={(v) => this.props.setChartStatus(!v)}
+                chartType={chart_type}
+                isMobile={is_mobile}
+                granularity={granularity}
+                requestAPI={wsSendRequest}
+                requestForget={wsForget}
+                requestForgetStream={wsForgetStream}
+                requestSubscribe={wsSubscribe}
+                settings={settings}
+                symbol={symbol}
+                // topWidgets={ChartTopWidgets}
+                isConnectionOpened={is_socket_opened}
+            // clearChart={false}
+            // importedLayout={chart_layout}
+            // onExportLayout={this.props.exportLayout}
+            // shouldFetchTradingTimes={!this.props.end_epoch}
+            >
+                {/* <ChartMarkers /> */}
+            </SmartChart>
+        );
+    }
+}
+
+export default connect(({ chart_store, common, ui }) => ({
+    is_mobile        : ui.is_mobile,
+    is_socket_opened : common.is_socket_opened,
+    updateChartType  : chart_store.updateChartType,
+    updateGranularity: chart_store.updateGranularity,
+    granularity      : chart_store.granularity,
+    chart_type       : chart_store.chart_type,
+    settings         : {
+        assetInformation            : false, // ui.is_chart_asset_info_visible,
+        countdown                   : true,
+        isHighestLowestMarkerEnabled: false, // TODO: Pending UI,
+        lang                        : common.current_language,
+        position                    : ui.is_chart_layout_default ? 'bottom' : 'left',
+        theme                       : ui.is_dark_mode_on ? 'dark' : 'light',
```

Comment:
Please confirm this won't cause any issues when switching from dark trader app to our own, if it does we may need to hardcode it to `false` until we have our dark theme ready.